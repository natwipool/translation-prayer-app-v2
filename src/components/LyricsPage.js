import React from 'react';
import LyricsList from './LyricsList';

const LyricsPage = (props) => (
  <div>
    <LyricsList 
      playlist={props.playlists.find((playlist, index) =>
        index === props.index
      )}
    />
  </div>
);

export default LyricsPage;