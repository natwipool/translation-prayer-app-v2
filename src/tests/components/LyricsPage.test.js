import React from 'react';
import { shallow } from 'enzyme';
import LyricsPage from '../../components/LyricsPage';
import transPrayers from '../fixtures/transPrayers';

test('should render LyricsPage correctly', () => {
  const wrapper = shallow(<LyricsPage
    playlists={transPrayers}
    index={0}
  />)
  expect(wrapper).toMatchSnapshot();
});