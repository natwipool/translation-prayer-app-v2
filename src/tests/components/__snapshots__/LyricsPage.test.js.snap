// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LyricsPage correctly 1`] = `
<div>
  <Connect(LyricsList)
    playlist={
      Object {
        "category": "บทสวดมนต์ภาคเย็น",
        "duration": 155.99425,
        "filename": "11_เขมาเขมสรณคมนปริทีปิกาคาถา.mp3",
        "id": "11",
        "lyrics": Array [
          "abc",
          "dfg",
        ],
        "precept": "เขมาเขมสรณคมนปริทีปิกาคาถา",
      }
    }
  />
</div>
`;
